src/pages/Clusters.js

```diff
@@ -64,7 +64,7 @@ const Clusters = () => {
   const clustersByProject = _.groupBy('googleProject', clusters)
 
   return h(Fragment, [
-    h(TopBar, { title: 'Notebook runtimes', href: Nav.getLink('clusters') }),
+    h(TopBar, { title: 'Notebook Runtimes', href: Nav.getLink('clusters') }), // FIXME: what about this one?
```

Comment:
See below comment for why this is a FIXME

Reply:
I think we can leave the URL as-is, so this should not change.